README.md

```diff
@@ -72,6 +72,17 @@ init({
 captureMessage('Hello, world!');
 ```
 
+If you want sentry to be customized for the browsers you want to support use the `esm` build:
```

Comment:
This sentence (and the one below) sounds kinda confusing. I'd reword it somehow.

Reply:
Sure I didn't come up with a better one, but if you have a better idea